similar/main/net_udp.cpp

```diff
@@ -3820,7 +3820,7 @@ int net_udp_setup_game()
 		Netgame.Tracker = 0;
 #endif
 
-	return i >= 0;
+	return (i >= 0) ? window_event_result::close : window_event_result::handled;
```

Comment:
I do not find it to be more clear, but it does not bother me.  I was surprised to see that, as our lead clang user, you were adding code that clang warns about. :)